.canon.yaml

```diff
@@ -1,14 +1,15 @@
-rtsp:
+viam-rtsp:
```

Comment:
Updated to follow the pattern in RDK .canon.yaml & to update to use images that have go 1.21